$OpenBSD: patch-src_wmapp_cc,v 1.10 2018/04/08 19:10:10 jasper Exp $

error: invalid suffix on literal; C++11 requires a space between literal and identifier

Index: src/wmapp.cc
--- src/wmapp.cc.orig
+++ src/wmapp.cc
@@ -221,7 +221,7 @@ static void registerProtocols2(Window xid) {
 #endif
 
     long pid = getpid();
-    const char wmname[] = "IceWM "VERSION" ("HOSTOS"/"HOSTCPU")";
+    const char wmname[] = "IceWM " VERSION " (" HOSTOS "/" HOSTCPU ")";
 
 #ifdef GNOME1_HINTS
     XChangeProperty(xapp->display(), xid, 
@@ -338,7 +338,7 @@ static void initFontPath(IApp *app) {
             char ** fontPath(XGetFontPath(xapp->display(), &ndirs));
 
             char ** newFontPath = new char *[ndirs + 1];
-            newFontPath[ndirs] = fontsdir;
+            newFontPath[ndirs] = (char *)fontsdir;
 
             if (fontPath)
                 memcpy(newFontPath, fontPath, ndirs * sizeof (char *));
